skrifa/src/provider.rs

```diff
@@ -47,6 +48,11 @@ pub trait MetadataProvider<'a>: raw::TableProvider<'a> + Sized {
     fn charmap(&self) -> Charmap<'a> {
         Charmap::new(self)
     }
+
+    /// Returns the collection of color outlines.
+    fn color_outlines(&self) -> ColorOutlineCollection<'a> {
```

Comment:
Yes, I think this works as a good entry point. And is perhaps a bit more idiomatic than just a "do_traverse_paint" function that takes a glyph id somewhere. It also answers the "what glyph format is available" question.

Reply:
Thanks. I’m trying to keep the API as consistent as possible. Fonts don’t make it easy 🤷